.ci.yaml

```diff
@@ -2084,6 +2084,16 @@ targets:
       task_name: picture_cache_perf__timeline_summary
     scheduler: luci
 
+  - name: Linux_android picture_cache_complexity_scoring_perf__timeline_summary
```

Comment:
Add `    bringup: true` as one of the first two property lines on new benchmarks. The bots will monitor the flaky status and automatically submit a deflaking PR when the consistency is established.

Reply:
(I see @keyonghan already mentioned this while I was writing my comments.)